lib/screen/home_screen.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_training/screen/weather_screen.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({super.key});
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  @override
+  void initState() {
+    super.initState();
+
+    WidgetsBinding.instance.endOfFrame.then((_) => _loadScreen());
+  }
+
+  Future<void> _loadScreen() async {
+    await Future.delayed(const Duration(milliseconds: 500), () {});
```

Comment:
[nits]
細かいですが、await を使った方法だと二つ目の引数は省略できた気がします！

```suggestion
    await Future.delayed(const Duration(milliseconds: 500));
```

Reply:
なるほど！実は、引数を外してCommit・Pushしようと思ったのですが、
```
The type argument(s) of the constructor 'Future.delayed' can't be inferred.  Use explicit type argument(s) for 'Future.delayed'.
```
とWarningが出てしまい、warningを消すためにあえて二つ目の引数も記載しています！

Reply:
@iseruuuuu 
あ〜、そういえば、void を指定しないといけないんでした！
失礼しました 🙇 

```suggestion
    await Future<void>.delayed(const Duration(milliseconds: 500));
```

Reply:
教えてくれた通りにやってみたらWarningが消えました。これで行こうと思います。ここについては詳細やWarinigの意味などがまだよくわかっていないのでこれらをしっかりと理解した上で、進めていこうと思います！！！